### ISSUE TITLE
Gephi fails to load .gephi file, without notification

### ISSUE THREAD

**Issue by GrantRVD** [gephi/gephi#2263]
## Expected Behavior
Gephi should be able to load its saved data, or provide more details on what the issue is.

## Current Behavior
Gephi fails to load data without any obvious error message. Only message is found on the Output window, which isn't open by default.

## Possible Solution
Valid format when saving file, or provide easy tool for converting unloadable data into a different format.

Behavior always happens after an OpenGL error causes the Overview window to become unusable, so it may also solve the issue to make it possible to restart Gephi's link to OpenGL without closing the software.

## Steps to Reproduce

1. Collect Full Twitter Network with the Twitter Streaming Importer
2. Once network has between 10k and 100k edges, try using a filter (Giant Component, or Atrribute>Equal) until Overview window freezes. May also be accomplished by trying to use the timeline player.
3. With Overview window frozen but the rest of Gephi still working, save the current workspace.
4. Quit Gephi
5. Restart Gephi
6. Try to load the previous workspace from the corresponding `.gephi` file
7. Check the Output window for lines that look like this
```
[SEVERE] null 
org.gephi.project.io.GephiFormatException: Gephi failed importing the project.

Exception: ArrayIndexOutOfBoundsException : 21904
File: org.gephi.graph.impl.Serialization
Line : 950
	at org.gephi.project.io.LoadTask.run(LoadTask.java:173)
	at org.gephi.desktop.project.ProjectControllerUIImpl$7.run(ProjectControllerUIImpl.java:341)
	at org.gephi.utils.longtask.api.LongTaskExecutor$RunningLongTask.run(LongTaskExecutor.java:274)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ArrayIndexOutOfBoundsException: 21904
	at org.gephi.graph.impl.Serialization.deserializeTimestampIndexStore(Serialization.java:950)
	at org.gephi.graph.impl.Serialization.deserialize(Serialization.java:2104)
	at org.gephi.graph.impl.Serialization.deserializeTimeStore(Serialization.java:1066)
	at org.gephi.graph.impl.Serialization.deserialize(Serialization.java:2119)
	at org.gephi.graph.impl.Serialization.deserializeGraphStore(Serialization.java:323)
	at org.gephi.graph.impl.Serialization.deserialize(Serialization.java:2062)
	at org.gephi.graph.impl.Serialization.deserializeGraphModel(Serialization.java:245)
	at org.gephi.graph.api.GraphModel$Serialization.read(GraphModel.java:135)
	at org.gephi.graph.GraphPersistenceProvider.readBytes(GraphPersistenceProvider.java:88)
	at org.gephi.project.io.LoadTask.readWorkspaceChildrenBytes(LoadTask.java:327)
	at org.gephi.project.io.LoadTask.run(LoadTask.java:130)
	... 7 more

[WARNING] Services/AutoupdateType/org_gephi_branding_update_center.instance: url_key attribute deprecated in favor of url 
[WARNING] Services/AutoupdateType/org_gephi_branding_update_center_1.instance: url_key attribute deprecated in favor of url 
```

## Context
I'm trying to analyze Twitter networks around highly sensitive news items by collecting a sufficient number of Tweets, Users, etc. over time. This issue interfered with two attempts to perform an analysis, the first of which had been running for 12+ hours, the second running for almost 4. There appears to be no way to recover the data, and since the Streaming importer only gets data as they arrive, I cannot reacquire the data lost in the corrupted files.
## Your Environment

* Version used: Gephi 0.9.2
* Java version: openjdk-8
* Operating System: Ubuntu 20.04

**Comment by mbastian** [comment]
Hi @GrantRVD I've tried to reproduce this issue but without luck so far. If you still have a corrupted file handy I would appreciate if you could send it to me at mathieu.bastian [at] gephi.org

Thank you!

**Comment by GrantRVD** [comment]
Hi there! Sorry for not seeing this reply sooner - GitHub notifications are apparently off.

I'm in the middle of replacing some damaged hard drives (I'm sensing a theme with my issues...), but if I find any of the files I couldn't reload, I'll be sure to send it along.

**Comment by mbastian** [comment]
No worries @GrantRVD :) I'm closing the issue then. We've made some patches to the IO system to make it more robust already so hopefully this one got addressed too. Those will be included in the upcoming 0.9.3 release. Feel free to reopen when needed.
